app/values/location.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 class Location
-  FIRESTONE_CODES = ["mss", "hsvm", "rcpxm"].freeze
-  MUDD_CODES = ["mudd", "prnc", "rcpph", "oo", "sc", "sls"].freeze
```

Comment:
Everything was updated except "oo", which was removed. Just double checking that this was intentional. If so, everything else looks good to me.

Reply:
Thanks @sdellis ! Yes, that was intentional.

Reply:
Can we deploy this to abid-staging first please to test it?